projection/src/test/scala/org/apache/pekko/projection/r2dbc/TestConfig.scala

```diff
@@ -71,6 +71,9 @@ object TestConfig {
       }
     }
     pekko.actor.testkit.typed.default-timeout = 10s
-    """)))
+    """))
   }
+
+  // FIXME ideally every dependant that combines this config with other configs should load/resolve at their callsites
```

Comment:
I don't understand this comment. Dependent is the correct spelling. If you don't intend to fix this, could it be made into a GitHub issue and you can even put its URL here?

Reply:
Idea is that configs should first be merged, then resolved. Current `main` branch code resolves `TestConfig`, then uses it as fallback in test overrides, and if test needs to override any substitutions it is unable, because they were already resolved.

It felt weird to me to create an issue for it because it is kind of a test hygiene thing that only maintainers might somewhat care about, but I wanted to at least mention it. Do you think this is a valid improvement that warrants an issue or should this comment just be removed?

> Dependent is the correct spelling.

IIUC this is a UK/US English thing, but I can of course fix.

Reply:
Dependent is correct here.
https://en.wiktionary.org/wiki/dependent

Dependant is a different word in UK English to describe a child or someone else who depends on their parents or guardians. I think Americans spell this Dependent.
https://en.wiktionary.org/wiki/dependant